### DISCUSSION TITLE
Premières analyses de la données des DUMPS - titres

### DISCUSSION THREAD

**Discussion by GenFab28** [culturecreates/artsdata-planet-lavitrine#31] [Modelling discussions]
Alors, nous remarquons que certains titres s'affichent comme suit :

![Capture d’écran 2024-05-16 103352](https://github.com/culturecreates/artsdata-planet-lavitrine/assets/116387423/b53a95e6-9a14-4019-b1dc-193c9751030f)

Est-ce possible de retirer Conseil de la culture où il y a réellement ce titre dans la BD moissonnée?

@saumier  @christianroy

**Comment by saumier** [comment]
Those titles are really scraped from the webpage html. We have a mechanism to choose different places on the webpage. I will pass the request to @tammy-culture. The crawler usually tries to remove it so I am not sure why it does not in this case. I took a quick look at the webpage by clicking the URI and then clicking webpage link in Footlight console. It looks very doable.

**Reply by tammy-culture** [reply]
On it.
